changelogs/fragments/7374-fix-selective-callback-taskname-length.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "selective callback plugin - fix length of task name lines in output always being 3 characters longer than desired."
```

Please add a reference to the ticket as in:
```suggestion
  - "selective callback plugin - fix length of task name lines in output always being 3 characters longer than desired (https://github.com/ansible-collections/community.general/pull/7374)."
```

---

Done